### PR TITLE
fix: preview-img style

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/38636

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
https://ant.design/docs/spec/data-entry-cn#%E5%BC%80%E5%85%B3%EF%BC%88Switch%EF%BC%89

https://preview-38662-ant-design.surge.sh/docs/spec/data-entry-cn#%E5%BC%80%E5%85%B3switch

这里还不对。

图片

**Review by li-jia-nan — APPROVED** [review]

**Comment by afc163** [comment]
还有文字和图片背景都出不来。

图片

图片

图片

**Comment by MadCcc** [comment]
> 还有文字和图片背景都出不来。
> 
> 图片
> 
> 图片
> 
> 图片

这些要上 rehype 搞了，有点复杂。

**Comment by afc163** [comment]
图片
好，今天不搞了，这个样式可以先加上。

File: .dumi/theme/common/GlobalStyles.tsx [hunk 1]
```diff
@@ -1919,6 +1919,8 @@ const GlobalStyles = () => {
   clear: both;
   float: right;
   margin: 0 0 70px 64px;
+  background-color: #eee;
+  padding: 24px;
```
Comment (afc163) [inline .dumi/theme/common/GlobalStyles.tsx:1923]: 别抄我的截图，按 4.x 的样式来 https://ant.design/docs/spec/proximity-cn

**Comment by MadCcc** [comment]
image
gan jue ke yi de ya感觉可以的呀

**Pull request merged**